src/rustup-cli/rustup_mode.rs

```diff
@@ -788,6 +792,14 @@ fn doc(cfg: &Cfg, m: &ArgMatches) -> Result<()> {
         "index.html"
     };
 
+    if m.is_present("print-only") {
+        let path =  toolchain.doc_path(doc_url);
+        match path {
+            Ok(v) => return Ok(println!("file:/{}", v.to_str().unwrap())),
```

Comment:
This probably wants to be `v.display()`.

Reply:
Cleared the extra files, BTW v.display() for some reason doesn't provide the exact URL. So didn't change that part. Thanks.